src/fu-debug.c

```diff
@@ -82,20 +83,22 @@ fu_debug_handler_cb (const gchar *log_domain,
 				       g_date_time_get_microsecond (dt) / 1000);
 	}
 
-	/* each file should have set this */
-	if (log_domain == NULL)
-		log_domain = "FIXME";
-
 	/* pad out domain */
-	domain = g_string_new (log_domain);
-	for (gsize i = domain->len; i < 20; i++)
-		g_string_append (domain, " ");
+	if (!self->no_domain) {
+		/* each file should have set this */
+		if (log_domain == NULL)
+			log_domain = "FIXME";
```

Comment:
While touching this, it makes me wonder - should this be an assertion rather than hacking together a fixup?

Reply:
Maybe something in CI then we should try to catch any FIXME domains that we forgot.

Reply:
Probably not too important, it's in meson.build and we'll catch during development.